application-example.conf

```diff
@@ -46,10 +46,28 @@ elastic {
   sendEnabled: false
 }
 
+# Solr default mode
 solr {
   url: "http://localhost:8983/solr/callbacktest"
 }
 
+# Solr cloud mode with url
```

Comment:
Should comment out the second two solr examples so there's only one that's active
This conf file tries to be an example of a valid configuration that someone could actually use
The file has elastic, solr, and opensearch configs alongside each other, but that's OK because the one that's used is determined by the indexer type (which we're not specifying explicitly here -- this file needs a deeper review -- backlog ticket needed for that)
Wouldn't want to have multiple solr blocks in the same config though as they could collide -- so keep the first one uncommented and comment the other two.

Reply:
commented out and added a ticket [LC-338](https://kmwllc.atlassian.net/browse/LC-338)